packages/okta-angular/package.json

```diff
@@ -8,19 +8,18 @@
     "dist"
   ],
   "scripts": {
+    "build:dependencies": "yarn ngc && cd test/e2e/harness && yarn install",
```

Comment:
For context - we previously ran `yarn pack` to create a `*.tgz` of our `@okta/okta-angular` package, then would add it directly to our test-harness via `yarn add` (`npm install --save`).

This is unnecessary, as we can reference the `package.json` in the library via the relative install path (shown [here](https://github.com/okta/okta-oidc-js/pull/317/files#diff-37c86cdb494a08f771023602f8e2f496R25)).